src/DSharp.Compiler/Compiler/ExpressionBuilder.cs

```diff
@@ -742,6 +762,39 @@ private Expression ProcessDotExpressionNode(BinaryExpressionNode node)
                 objectExpression = ProcessDotExpressionNode(node, filter, out memberSymbol);
             }
 
+            //If we are unable to parse the right hand child as a local member of the left hand expression,
+            //then we want to try and find out if the token is a Extension method usage.
+            if (objectExpression == null)
+            {
+                if (node.LeftChild.Token is LiteralToken leftLiteralToken)
+                {
+                    string typeName = leftLiteralToken.LiteralType == LiteralTokenType.String
```

Comment:
this is like a hack, right? we can have many types of literals, not only strings. Can you drop an explanation on why this is needed?

Reply:
This might actually be a work around for something that's no longer an issue. lemme try some stuff